patches/browserify-sign++readable-stream+2.3.8.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/node_modules/browserify-sign/node_modules/readable-stream/lib/_stream_writable.js b/node_modules/browserify-sign/node_modules/readable-stream/lib/_stream_writable.js
+index e1e897f..1e2c65f 100644
+--- a/node_modules/browserify-sign/node_modules/readable-stream/lib/_stream_writable.js
++++ b/node_modules/browserify-sign/node_modules/readable-stream/lib/_stream_writable.js
+@@ -54,7 +54,10 @@ function CorkedRequest(state) {
+ /* </replacement> */
+
+ /*<replacement>*/
+-var asyncWrite = !process.browser && ['v0.10', 'v0.9.'].indexOf(process.version.slice(0, 5)) > -1 ? setImmediate : pna.nextTick;
++if (!process.version) {
++  console.warn('Empty process version in browserify-sign/readable-stream/_stream_writable.js');
++}
++var asyncWrite = !process.browser && ['v0.10', 'v0.9.'].indexOf(process.version?.slice(0, 5)) > -1 ? setImmediate : pna.nextTick;
```

Comment:
I've been unable to pinpoint why this sub-dependency cannot fetch the correct process version, so I added a safe check to avoid the crash and a console warning to remind us that this needs proper fixing.